File: C/Mid Codes/LargestNumber.c
//Find the largest number from 3 numbers
#include<stdio.h>

int main() {
    int a,b,c;
    printf("Input 3 Numbers: ");
    scanf("%d%d%d",&a,&b,&c);
    if (a>=b && a>=c)
        printf("%d is the largest number",a);
    else if (b>=c && b>=c)
        printf("%d is the largest number",b);
     else if (c>=a && c>=a)
        printf("%d is the largest number",c);

    return 0;
}

File: Java/Academic/Mid/LargestNumber.java
/* . Largest of Three Numbers Using Nested. Write a Java program that takes three integer inputs and determines which one is the largest using nested if statements.
 */

import java.util.Scanner;

public class LargestNumber {
    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        System.out.println("Enter 3 numbers");
        int a=in.nextInt();
        int b=in.nextInt();
        int c=in.nextInt();

        if(a>=b){
            if(a>=c){
                System.out.println("A is the Largest number");
            }
        }
        else if(b>=a){
            if(b>=c){
                System.out.println("B is the Largest number");
            }
        }
        else{
                System.out.println("C is the Largest number");
            }
    }
}
